client/checkout/blocks/style.scss

```diff
@@ -88,13 +104,14 @@ button.wcpay-stripelink-modal-trigger:hover {
 					justify-self: end;
 				}
 			}
-		}
 
-		.bnpl-message {
-			width: 100%;
+			&__pmme-container {
+				width: 100%;
+				pointer-events: none;
```

Comment:
Added to ensure that clicking on the Payment Method Messaging Element expands the payment method